spec/rubocop/cop/lint/suppressed_exception_spec.rb

```diff
@@ -51,6 +72,16 @@ def foo
       RUBY
     end
 
+    it 'does not register an offense for empty rescue with comment in `def`' do
+      expect_no_offenses(<<~RUBY)
+        def foo
+          do_something
+        rescue
+          # do nothing
+        end
+      RUBY
+    end
```

Comment:
```suggestion
    context 'with comment in `def`' do
      it 'does not register an offense for empty rescue' do
        expect_no_offenses(<<~RUBY)
          def foo
            do_something
          rescue
            # do nothing
          end
        RUBY
      end
    end
```

Reply:
> context 'with comment in `def`' do

Since it's a comment for `rescue` instead of `def`, I adjusted the context a bit :-)